server.js

```diff
@@ -0,0 +1,18 @@
+import Importer from './importer';
+import DirWatcher from './dirWatcher';
+
+const importer = new Importer();
+const watcher = new DirWatcher();
+
+watcher.watch('./data', 1000);
+watcher.on('dirwatcher:changed', (path, files) => {
+    const dataAsync = Promise.all(files.map(f => {
```

Comment:
Pretty strange part of code. You log `dataAsync` which is promise + there is no return from function that you pass to `map`.